resources/fetchersManager/data_test.go

```diff
@@ -15,11 +15,12 @@
 // specific language governing permissions and limitations
 // under the License.
 
-package manager
+package fetchersManager
```

Comment:
I prefer to revert the file.

I know that we might need to do it again in the future.

But we don't know when/how we will implement the validation - until then, I think we should not assume anything about the implementation.